code/modules/jobs/job_types/roguetown/refugees/types/ecclesiast.dm

```diff
@@ -96,34 +97,215 @@
+					H.verbs += list(/mob/living/carbon/human/proc/devotionreport, /mob/living/carbon/human/proc/clericpray)
+				if(/datum/patron/divine/dendor)
+					H.mind.adjust_skillrank(/datum/skill/combat/polearms, 2, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/misc/athletics, 2, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/magic/holy, 3, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 2, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 2, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/misc/reading, 2, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/misc/climbing, 1, TRUE)
 					H.mind.adjust_skillrank(/datum/skill/misc/sewing, 1, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/labor/farming, 4, TRUE)
 					H.mind.adjust_skillrank(/datum/skill/craft/cooking, 1, TRUE)
 					pants = /obj/item/clothing/under/roguetown/tights/black
 					shoes = /obj/item/clothing/shoes/roguetown/armor
 					gloves = /obj/item/clothing/gloves/roguetown/leather
 					belt = /obj/item/storage/belt/rogue/leather
 					shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt/random
-					armor = /obj/item/clothing/suit/roguetown/armor/chainmail/hauberk
-					wrists = /obj/item/clothing/wrists/roguetown/bracers/leather
+					armor = /obj/item/clothing/suit/roguetown/armor/plate/half/iron
+					neck = /obj/item/clothing/neck/roguetown/gorget
+					backl = /obj/item/storage/backpack/rogue/satchel
+					r_hand = /obj/item/rogueweapon/spear
+					beltl = /obj/item/storage/belt/rogue/pouch/coins/poor
+					H.change_stat("endurance", 3)
+					H.change_stat("constitution", 3)
+					ADD_TRAIT(H, TRAIT_MEDIUMARMOR, TRAIT_GENERIC)
+					var/datum/devotion/C = new /datum/devotion(H, H.patron)
+					C.grant_spells(H)
+					H.verbs += list(/mob/living/carbon/human/proc/devotionreport, /mob/living/carbon/human/proc/clericpray)
+				if(/datum/patron/divine/ravox)
+					H.mind.adjust_skillrank(/datum/skill/combat/swords, 4, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/misc/athletics, 2, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/magic/holy, 2, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 3, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 3, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/misc/reading, 1, TRUE)
+					H.mind.adjust_skillrank(/datum/skill/misc/climbing, 1, TRUE)
+					pants = /obj/item/clothing/under/roguetown/tights/black
+					shoes = /obj/item/clothing/shoes/roguetown/armor
+					gloves = /obj/item/clothing/gloves/roguetown/leather
+					belt = /obj/item/storage/belt/rogue/leather
+					shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt/random
+					armor = /obj/item/clothing/suit/roguetown/armor/plate/half/iron
+					neck = /obj/item/clothing/neck/roguetown/gorget
+					backl = /obj/item/storage/backpack/rogue/satchel
+					beltr = /obj/item/rogueweapon/sword/iron
+					beltl = /obj/item/storage/belt/rogue/pouch/coins/poor
+					H.change_stat("strength", 4)
```

Comment:
4 strength is an obscene stat buff for a preacher class. Thats on par with knights and templar. Certainly not.